Jenkinsfile

```diff
@@ -17,16 +17,17 @@ pipeline {
       when { not { buildingTag() } }
       steps {
         script {
-          target = "install"
+          target = "verify"
         }
       }
     }
     stage('Setup release') {
       when { buildingTag(); not { changeRequest() }  }
       steps {
         script {
-          target = "install deploy -P release  --settings settings.xml"
+          target = "verify deploy -P release  --settings settings.xml"
```

Comment:
The `verify deploy` is redundant as `deploy` will execute all lifecycle steps up to it:

https://maven.apache.org/guides/introduction/introduction-to-the-lifecycle.html

Only `clean` and `site` are alternate/independent lifecycles.